src/Silex/EventListener/StringToResponseListener.php

```diff
@@ -23,12 +24,8 @@
  */
 class StringToResponseListener implements EventSubscriberInterface
 {
-    /**
-     * Handles string responses.
-     *
-     * @param GetResponseForControllerResultEvent $event The event to handle
-     */
-    public function onKernelView(GetResponseForControllerResultEvent $event)
+
+    public function onKernelView(ViewEvent $event)
```

Comment:
From: https://github.com/symfony/http-kernel/blob/v4.4.50/Event/GetResponseForControllerResultEvent.php

```
/**
 * @deprecated since Symfony 4.3, use ViewEvent instead
 */
```